### PR TITLE
add regressive, fix tcoffee and consensus

### PR DESCRIPTION
**Pull request by luisas**
## PR checklist

Add regressive module
Update T-coffee module - remove the fix that was implemented for the regressive for compression, since now it is its own module. 

Update consensus module to also provide evaluation outputs 

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [x] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [x] Add a resource `label`
- [x] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - For modules:
    - [x] `nf-core modules test <MODULE> --profile docker`
    - [x] `nf-core modules test <MODULE> --profile singularity`
    - [x] `nf-core modules test <MODULE> --profile conda`
  - For subworkflows:
    - [x] `nf-core subworkflows test <SUBWORKFLOW> --profile docker`
    - [x] `nf-core subworkflows test <SUBWORKFLOW> --profile singularity`
    - [x] `nf-core subworkflows test <SUBWORKFLOW> --profile conda`

### CODE REVIEW

File: modules/nf-core/tcoffee/regressive/main.nf [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def tree_args = tree ? "-regtree $tree" : ""
+    def template_args = template ? "-template_file $template" : ""
+    def outfile = compress ? "stdout" : "${prefix}.aln"
```
Comment (itrujnara) [inline modules/nf-core/tcoffee/regressive/main.nf:28]: Equal sign alignment

File: modules/nf-core/tcoffee/regressive/meta.yml [hunk 2]
```diff
@@ -0,0 +1,80 @@
+name: "tcoffee_regressive"
+description: Aligns sequences using T_COFFEE
```
Comment (itrujnara) [inline modules/nf-core/tcoffee/regressive/meta.yml:2]: Make it more descriptive, this is a specific algorithm

**Review by itrujnara — COMMENTED** [review]
Minor comments

File: modules/nf-core/tcoffee/regressive/meta.yml (hunk 2 above)
Comment (luisas) [reply modules/nf-core/tcoffee/regressive/meta.yml]: ok @itrujnara, done! Thanks for the review

**Review by itrujnara — APPROVED** [review]
LGTM, just fix the CI issues

**Pull request merged**